website/docs/docs/use-dbt-semantic-layer/sl-faqs.md

```diff
@@ -66,6 +66,13 @@ The dbt Semantic Layer measures usage in distinct 'Queried Metrics'. Refer to th
 
 ## Availability
 
+<expandable alt_header="What data platforms are supported by the dbt Semantic Layer?">
+
+The dbt Semantic Layer supports the following data platforms: Snowflake, BigQuery, Databricks, or Redshift.
+
+Support for other data platforms, such as Fabric and Trino, isn't available at this time. If you're interested in using the dbt Semantic Layer with a different data platform, please reach out to us [here](https://www.getdbt.com/get-started).
```

Comment:
Maybe we don't need to call out platforms we *don't* support.
I don't think that link is going to be helpful as a contact link. Is there something more direct we could use like and email address or a link to a page with contact info?

```suggestion
If you're interested in using the dbt Semantic Layer with a data platform not on the list, please [contact us](https://www.getdbt.com/get-started).
```

Reply:
i think it's helpful to be explicit and not encourage inferring, personally imho. esp since users/sa's asked about it. 

i like the mesh of:
> Support for other data platforms, such as Fabric and Trino, isn't available at this time. If you're interested in using the dbt Semantic Layer with a data platform not on the list, please [contact us](https://www.getdbt.com/get-started).